components/local-app/cmd/workspace-ssh.go

```diff
@@ -15,7 +15,17 @@ var dryRun bool
 var workspaceSSHCmd = &cobra.Command{
 	Use:   "ssh <workspace-id>",
 	Short: "Connects to a workspace via SSH",
-	Args:  cobra.ExactArgs(1),
+	Args:  cobra.MinimumNArgs(1),
+	Example: `  # connect to workspace with current terminal session
+  $ gitpod workspace ssh <workspace-id>
+
+  # Execute with ssh command
+  $ gitpod workspace ssh <workspace-id> -- ls -la
+  $ gitpod ws ssh <workspace-id> -- -t watch date
+
+  # Get all SSH features with --dry-run
+  $ $(gitpod workspace ssh <workspace-id> --dry-run) -- ls -la
```

Comment:
i still we should remove, wait for bugs, and fix them properly, not spread hacks.